### DISCUSSION TITLE
RISOLTO-Ripristino backup Hassio RPI4 su MiniPC

### DISCUSSION THREAD

**Discussion by Stomodine** [indomus/forum#2273] [Hardware & Firmware]
Ciao

al momento ho un RPI4 sul quale gira Hassio (su SSD esterno) ma vorrei passare su MiniPC per avere prestazioni migliori con MotionEye; posso semplicemente fare il ripristino del backup sul nuovo HW o devo adattare qualcosa?

**Comment by snakuzzo** [comment]
se pensi di usare HA OS o HA Supervised sul nuovo mini PC, devi semplicemente installare e poi ripristinare il backup.

**Reply by Stomodine** [reply]
ottimo grazie
